diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/rules/CustomGetterSetterRule.kt

```diff
@@ -0,0 +1,42 @@
+package org.cqfn.diktat.ruleset.rules
+
+import com.pinterest.ktlint.core.Rule
+import com.pinterest.ktlint.core.ast.ElementType.GET_KEYWORD
+import com.pinterest.ktlint.core.ast.ElementType.PROPERTY_ACCESSOR
+import com.pinterest.ktlint.core.ast.ElementType.SET_KEYWORD
+import org.cqfn.diktat.common.config.rules.RulesConfig
+import org.cqfn.diktat.ruleset.constants.Warnings
+import org.cqfn.diktat.ruleset.utils.*
+import org.jetbrains.kotlin.com.intellij.lang.ASTNode
+
+/**
+ * Inspection that checks that no custom getters and setters are used for properties.
+ */
+class CustomGetterSetterRule(private val configRules: List<RulesConfig>) : Rule("custom-getter-setter") {
```

Comment:
Should this rule be so strict by design or can be enhance it with some exceptions? Like empty private setters that make a property read-only might be legal.

Reply:
btw - it's a good idea, may be we should reflect it in our code style